src/examples/radio/RadioExamples.tsx

```diff
@@ -0,0 +1,75 @@
+import { GoARadioGroup, GoARadioItem, GoAFormItem, GoAInput } from "@abgov/react-components";
+import { Sandbox } from "@components/sandbox";
+import { CodeSnippet } from "@components/code-snippet/CodeSnippet.tsx";
+
+const noop = () => { };
+
+export default function CheckboxExamples () {
+  return (
+    <>      
+      <h2 id="component-examples" className="hidden" aria-hidden="true">Examples</h2>
+      <h3 id="component-example-expand-collapse-form">Use a component as a description</h3>
+      <Sandbox fullWidth skipRender>
+        {/*Angular*/}
+        <CodeSnippet
+        lang="typescript"
+        tags="angular"
+        allowCopy={true}
+        code={`
+              <goa-form-item label="Select one option">
+              <goa-radio-group name="contact" value="1" (_change)="onChange($event)">
+                <goa-radio-item value="1" label="Option 1">
+                  <div slot="description">
+                    <goa-form-item mt="m" label="Input label">
+                      <goa-input name="phoneNumber" (_change)="onChange($event)" />
+                    </goa-form-item>
```

Comment:
@lizhuomeng71 Got it! I've replaced the stray React component with an Angular one in the Angular sample code.

Reply:
Same thing here for the text based description.